pom.xml

```diff
@@ -75,7 +75,7 @@
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
                 <artifactId>bom-2.346.x</artifactId>
-                <version>1478.v81d3dc4f9a_43</version>
+                <version>1556.vfc6a_f216e3c6</version>
```

Comment:
I guess it should now suffice to pick up https://github.com/jenkinsci/bom/pull/1607 as in #634?

Reply:
> I guess it should now suffice to pick up [jenkinsci/bom#1607](https://github.com/jenkinsci/bom/pull/1607) as in #634?

Yes.  For my own testing use, I created #633 as a draft pull request.  I confirmed that the originally reported issue is fixed and experimented with one branch strategy while investigating a null pointer exception in git plugin 4.14.0.  The null pointer exception is fixed in [git plugin 4.14.1](https://github.com/jenkinsci/git-plugin/releases/tag/git-4.14.1).